RecoPPS/Local/src/CTPPSDiamondRecHitProducerAlgorithm.cc

```diff
@@ -44,8 +45,17 @@ void CTPPSDiamondRecHitProducerAlgorithm::build(const CTPPSGeometry& geom,
     const float y_width = 2.0 * diamondDimensions.yHalfWidth;
     const float z_width = 2.0 * diamondDimensions.zHalfWidth;
 
-    // retrieve the timing calibration part for this channel
     const int sector = detid.arm(), station = detid.station(), plane = detid.plane(), channel = detid.channel();
+    //LUT calibration
+    std::vector<double> LUT;
```

Comment:
local names should start with a lower case

Reply:
done!